apps/api/src/app/events/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -140,6 +150,26 @@ export class SendMessageInApp extends SendMessageType {
       { read: false }
     );
 
+    await this.createExecutionDetails.execute(
+      CreateExecutionDetailsCommand.create({
+        environmentId: command.environmentId,
+        organizationId: command.organizationId,
+        subscriberId: command.subscriberId,
+        jobId: command.jobId,
+        notificationId: notification._id,
+        notificationTemplateId: notification._templateId,
+        messageId: message._id,
+        providerId: 'in-app',
```

Comment:
@scopsy, @p-fernandez, @ainouzgali should we do like this or should we allow it to be empty?

Reply:
If you mean `providerId` seeing how we are using it, I think it shouldn't be empty. Is there any case we might need the flexibility of being empty?

Reply:
yes because our in app notifications are our own system that provides so we do not have provider id for it...

Reply:
What about create different providers in house? To have something like:
```
enum NovuProviders {
   NOVU_IN_APP = 'novu-in-app',
   NOVU_EMAIL = 'novu-email',
   ...
}
```
So in case we ever become also a provider there wouldn't be a need to do any kind of migration. Also solving this thing for the future.

Reply:
I like the inhouse identifiers. It makes a lot of sense, definitely see a point in the future where we will differentiate between our own in app provider and another vendor integration 🎉